mrbgem.rake

```diff
@@ -20,42 +17,49 @@ MRuby::Gem::Specification.new('mruby-yaml') do |spec|
     end
   end
 
-  FileUtils.mkdir_p build_dir
+  use_system_library = ENV.fetch('MRUBY_YAML_USE_SYSTEM_LIBRARY', '') != ''
+
+  unless use_system_library
+    yaml_version = "0.2.1"
```

Comment:
Is the version downgrade necessary?

Reply:
No, this change was committed by mistake.